jvcl/run/JvMenus.pas

```diff
@@ -1255,7 +1255,7 @@ procedure TJvMainMenu.WMDrawItem(var Msg: TWMDrawItem);
         SaveIndex := SaveDC(hDC);
         try
           Canvas.Handle := hDC;
-          SetDefaultMenuFont(Canvas.Font);
+          SetDefaultMenuFont(TScreen.MenuFont);
```

Comment:
Does this compile with all Delphi versions? I'm wondering if this was available in D6 for instance

Reply:
Well, ok, `MenuFont` exists on `TScreen` but not as a class property, the code should really be `Screen.MenuFont` (notice the missing T at the beginning)